File: _old_CmdMents/Idolatry/BurnIdolatrousCity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.Idolatry
{
    /// <summary>
    /// Burn to the ground any city that has fallen into idolatry.
    /// </summary>
    /// <author>NET</author>
    class BurnIdolatrousCity : NoIncitingACityToIdolatry
    {

        public BurnIdolatrousCity()
        {
            base.AlternateText = null;
            base.Book = CommandmentBook.Deuteronomy;
            base.Chapter = 13;
            base.Verse = 16;
            base.ShortSummary = "Burn an idolatrous city to the ground.";
            base.Text = "\"'Gather all the plunder of the town into the middle of the public square and completely burn the town and all its plunder as a whole burnt offering to the LORD your God. It is to remain a ruin forever, never to be rebuilt.";

            base.CanBeCarriedOutToday = true;           // Civic duty only, however
            base.CanBeCarriedOutOnlyInIsrael = true;    // See above
            base.FollowedByChristians = CommandmentObedience.None;
            base.FollowedByMessianics = CommandmentObedience.Recognized;
            base.FollowedByObservantJews = CommandmentObedience.Attempted;      // Not at all certain of this. 
                                                                                // Judah's note: I agree that ultra-orthodox would like to follow this, and if they had sufficient political power, would actually carry this out. I agree with your labeling as Attempted.

            base.CommandmentType = CommandmentType.Positive;
            base.Number = 33;
        }

    }
}

File: EtzMitzvot/Commandments/Idolatry/BurnIdolatrousCity.ts
///<reference path='../commandmentBase.ts' />

class BurnIdolatrousCity extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Burn the city that has <br /> turned to idol worship.";
        this.text = "Gather all the plunder of the town into the middle of the public square and completely burn the town and all its plunder as a whole burnt offering to the LORD your God. It is to remain a ruin forever, never to be rebuilt.";
        this.book = TorahBook.Deuteronomy;
        this.chapter = 13;
        this.verse = 16;

        this.canBeCarriedOutToday = false;
        this.requiresLivingInIsrael = true;
        this.requiresTemple = false;
        this.commentary = null;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.None;
        this.messianicObservance = CommandmentObedience.RecognizedButPrevented;
        this.jewishObservance = CommandmentObedience.RecognizedButPrevented;
        this.number = 33;

        this.children = [
        ];
    }
}